pkg/kubelet/kubelet.go

```diff
@@ -1303,6 +1303,7 @@ func (kl *Kubelet) initializeModules() error {
 		collectors.NewVolumeStatsCollector(kl),
 		collectors.NewLogMetricsCollector(kl.StatsProvider.ListPodStats),
 	)
+	metrics.SetNodeName(kl.nodeName)
```

Comment:
can a node name be changed without a process restart? /cc @derekwaynecarr

Reply:
I do not find the place to change node name in code. But not sure.

Reply:
@brancz no.

Reply:
Thanks for the confirmation!